clients/venice-push-job/src/main/java/com/linkedin/venice/hadoop/VenicePushJob.java

```diff
@@ -2364,6 +2381,10 @@ void validateValueSchema(
         setting.storeName);
   }
 
+  boolean isUpdateSchema(String schemaString) {
```

Comment:
This check seems to be too simple.
Shall we send this update schema to the backend to check whether it matches any existing update schema or not?

Reply:
Ok, I can also add it, this is existing logic and we did not check it.

Reply:
Actually, after reading the code again I think we did validate the schema if the input schema is already a partial update schema. Existing logic try to locate the schema id from controller client call.